Automattic/jetpack
# Help center - add icon to front end admin bar

Fixes https://github.com/Automattic/dotcom-forge/issues/8466

This PR will load the help center on the front end of a site admin bar, however we need to limit when the help center will load to avoid issues where every logged in user on simple will see it. 

There is an incident where this broke the admin bar the last time we showed the help center on the front end - p58i-e0x-p2

To avoid this, we will only show the help center to users that are have `edit_posts` capability on the current site, otherwise the help center will not load.

## Proposed changes:


* Only show the help-center the logged in users that have the `edit_posts` capability.
* Updates the help center in `jetpack-mu-plugin' to show in the admin bar on the front end of a site

## Before

![Image](https://github.com/user-attachments/assets/0ada4fdc-7994-4589-92c8-41b16f6ffc81)

## After

<img width="967" alt="Screenshot 2024-07-31 at 18 03 12" src="https://github.com/user-attachments/assets/9b3296a3-e503-4bd6-b048-314b6d6ae4bc">

## Notes

I noticed that there is a z-index issue with the upgrade banner and the help center popover. I've created a PR on calypso to address this with https://github.com/Automattic/wp-calypso/pull/93156

<img width="717" alt="Screenshot 2024-07-31 at 18 03 23" src="https://github.com/user-attachments/assets/55684a98-5413-4f3f-bf19-efd53e9ae4a9">

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Follow test instructions below - https://github.com/Automattic/jetpack/pull/38651#issuecomment-2260948001
* Go to the wp-admin and confirm help center works as normal
* Go to the front end of the site and confirm help center works
